pyproject.toml

```diff
@@ -54,7 +54,7 @@ dependencies = [
     "pip-requirements-parser>=31.2.0",
     "prometheus-client>=0.10.0",
     "psutil",
-    "pydantic<3",
```

Comment:
did they have a breaking change in 2.9?

Reply:
Yes, and I've updated the implementation to work for pydantic 2.9, this limitation was removed.